doc/ref_model/chapters/chapter03.md

```diff
@@ -539,6 +542,150 @@ SyncE was standardized by the ITU-T, in cooperation with IEEE, as three recommen
+
+Kubernetes built in objects
+Pod and workloads | Description
+------------------|------------
+[Pod:](https://kubernetes.io/docs/concepts/workloads/pods/) | Pod is a collection of containers that can run on a host. This resource is created by clients and scheduled onto hosts.
+[ReplicaSet:](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/) | ReplicaSet ensures that a specified number of pod replicas are running at any given time.
+[Deployment:](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) | Deployment enables declarative updates for Pods and ReplicaSets.
+[DaemonSet:](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) | DaemonSet represents the configuration of a daemon set.
+[Job:](https://kubernetes.io/docs/concepts/workloads/controllers/job/) | Job represents the configuration of a single job.
+[CronJob:](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) | A CronJob manages time based Job, namely: once at a specified point in time repeatedly at a specified point in time
+[StatefulSet:](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) | StatefulSet represents a set of pods with consistent identities. Identities are defined as: network, storage. 
+
+Network objects | Description
+----------------|------------
+[Ingress:](https://kubernetes.io/docs/concepts/services-networking/ingress/) | Ingress is a collection of rules that allow inbound connections to reach the endpoints defined by a backend. An Ingress can be configured to give services externally-reachable urls, load balance traffic, terminate SSL, offer name based virtual hosting etc.
+[Service:](https://kubernetes.io/docs/concepts/services-networking/service/) | Service is a named abstraction of software service (for example, mysql) consisting of local port (for example 3306) that the proxy listens on, and the selector that determines which pods will answer requests sent through the proxy.
+[EndpointSlices:](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/) | Endpoints and Endpointslices are a collections of endpoints that contains the ip address, v4 and v6, of the pods that represents a service.
+[Network Policy:](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/) | NetworkPolicy defines which network traffic is allowed to ingress and egress from a set of pods.
+
+There is no need to explicitly define internal load balancers, servers pools, service monitors, firewalls and so on. The kubernetes sematics and relation between the different objects defined in the object manifests contains all the information needed.
+
+Example: The manifests for service *my-service* and the*deployment* with the four load balanced pods of type *my-app*
```

Comment:
I do not think that this level of detail adds any value to Anuket.

Reply:
OK< lets discuss this,